tests/phpunit/elementor/includes/settings/test-settings.php

```diff
@@ -1,12 +1,17 @@
 <?php
 namespace Elementor\Tests\Phpunit\Includes\Settings;
 
+use Elementor\Core\Admin\Options\Site_CSS_Print_Method;
+use Elementor\Plugin;
 use ElementorEditorTesting\Elementor_Test_Base;
 use Elementor\Core\Files\CSS\Global_CSS;
 
 class Test_Settings extends Elementor_Test_Base {
 
 	public function test_clear_css_cache_on_update_css_settings() {
+		// Register the option in order to listen to the changes.
+		Plugin::$instance->options->register( Site_CSS_Print_Method::class );
```

Comment:
Isn't it registered by default?

https://github.com/elementor/elementor/blob/ae434047f20f115377d41b957e7c73040bf3962c/core/admin/admin.php#L687

Reply:
It happens on the `admin.php`, but it seems that the test is running in another context